pkg/trace/stats/quantile/ddsketch_converter.go

```diff
@@ -0,0 +1,50 @@
+package quantile
+
+import (
+	"math"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
+)
+
+// DDSketchToGK only support contiguousBins and log mapping
+// for the moment
+func DDSketchToGK(ddSketch *pb.DDSketch) *SliceSummary {
+	gkSketch := SliceSummary{Entries: make([]Entry, 0, len(ddSketch.PositiveValues.ContiguousBinCounts))}
+	gamma := ddSketch.Mapping.Gamma
+
+	zeros := int(ddSketch.ZeroCount)
+	if zeros > 0 {
+		gkSketch.Entries = append(gkSketch.Entries, Entry{V: 0, G: zeros, Delta: 0})
+	}
+	indexOffset := int(ddSketch.PositiveValues.ContiguousBinIndexOffset)
+	var total int
+	for i, count := range ddSketch.PositiveValues.ContiguousBinCounts {
+		index := i + indexOffset - 1
+		g := int(count)
+		if g == 0 {
+			continue
+		}
+		total += g
+		v := valueFromIndex(index, gamma)
+
+		gkSketch.Entries = append(gkSketch.Entries, Entry{
+			V:     v,
+			G:     g,
+			Delta: g / 2,
```

Comment:
working place holder, 
I'll find a better value matching both current gk implem and t[his paper ](http://infolab.stanford.edu/~datar/courses/cs361a/papers/quantiles.pdf)